woocommerce/woocommerce-gutenberg-products-block
# Reviews by Product: add placeholders when loading reviews

### Screenshots

![Peek 2019-07-12 12-18](https://user-images.githubusercontent.com/3616980/61121286-2809d880-a49f-11e9-96a7-237964e28123.gif)

### How to test the changes in this Pull Request:

1. Create a post or page with a _Reviews by Product_ block, make it so there pagination (the _Load more_ button appears).
2. In the frontend, click on _Load more_ and verify a placeholder appears while loading.
3. Change the _Order by:_ select option and verify a placeholder appears as well while loading.
